src/packages/cli/src/bin.ts

```diff
@@ -198,6 +209,51 @@ async function main(): Promise<number> {
   return 0
 }
 
+/**
+ * Get a unique identifier for the project by hashing
+ * the directory with `schema.prisma`
+ */
+async function getProjectHash(): Promise<string> {
```

Comment:
return crypto... returns a string not a Promise?

Reply:
The function is async as it calls `getSchemaPath`